sdk/endpoints/batch/.net/EndpointClient.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Azure.Core;
+using Azure.ResourceManager;
+using Azure.ResourceManager.MachineLearning;
+using BatchInferencingSamples;
+using System.Text;
+using System.Text.Json;
+
+namespace batch_inferencing_samples
+{
+    internal class BatchEndpointActions
+    {
+        public static async Task<BatchEndpointData> Get(Guid subscriptionId, string resourceGroup, string workspaceName, string batchEndpointName, TokenCredential tokenCredential)
+        {
+            ResourceIdentifier batchEndpointResourceId =
+                new ResourceIdentifier($"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/batchEndpoints/{batchEndpointName}");
+
+            ArmClient armClient = new ArmClient(tokenCredential);
+            var batchEndpoint = armClient.GetBatchEndpointResource(batchEndpointResourceId);
+            var data = await batchEndpoint.GetAsync().ConfigureAwait(false);
+            return data.Value.Data;
+        }
+
+        public static async Task Invoke(Uri scoringUri, TokenCredential credentials)
+        {
+            var client = new HttpClient();
+
+            BatchScoringPayload payload = new BatchScoringPayload
+            {
+                Properties = new BatchScoringInputProperties
+                {
+                    InputData = new Dictionary<string, BatchScoringInput>
+                    {
+                        {
+                            "mnistinput",
+                            new BatchScoringInput
+                            {
+                                JobInputType = JobInputType.UriFolder,
+                                Uri = scoringUri,
```

Comment:
how come is the Uri set to `scoringUri`, isn't that suppoused to be pointing towards the datastore/dataset location ?

Reply:
Yeap changed tha tone!